src/main/java/org/assertj/core/api/AbstractThrowableAssert.java

```diff
@@ -119,6 +120,32 @@ public SELF hasCause(Throwable cause) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual {@code Throwable} has a cause the same as the given one, i.e. using == comparison
+   * <p>
+   * Example:
+   * <pre><code class='java'>
+   * Throwable invalidArgException = new IllegalArgumentException("invalid arg");
```

Comment:
join lines and let a space after `<code class='java'>` as in `   * <pre><code class='java'> Throwable`